content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx

```diff
@@ -234,6 +234,36 @@ const users = await prisma.user.findMany({
 
 See also: [Case sensitivity](case-sensitivity)
 
+### Filtering FAQs
+
+#### How does filtering work at the database level?
+
+For MySQL and PostgreSQL we are utilizing the [`LIKE`](https://www.w3schools.com/sql/sql_like.asp) (and [`ILIKE`](https://www.postgresql.org/docs/current/functions-matching.html#FUNCTIONS-LIKE)) operator to search for a given pattern. These operators have built-in pattern matching using symbols unique to `LIKE`. These include `%` for zero or more characters (similar to `*` in other regex implementations) and `_` for one character (similar to `.`).
+
+If you would like to match the literal characters `%` or `_`, make sure to escape those characters. For example:
+
+```ts
+const users = await prisma.user.findMany({
+  where: {
+    name: {
+      startsWith: '_',
+    },
+  },
+})
+```
+
+The above query will match any name that starts with any character. That is to say, any user where name is not `NULL`. If you instead wanted to find any user whose name starts with a literal `_` you could do
+
+```ts highlight=4
+const users = await prisma.user.findMany({
+  where: {
+    name: {
+      startsWith: '_', // note that the `_` character is escaped
```

Comment:
It would be valuable to have a more concrete example with a name

Reply:
fixed!